build.sh

```diff
@@ -28,6 +28,26 @@ installer_url () {
   echo "http://mirror.racket-lang.org/installers/${version}/${installer_path}";
 };
 
+build_7x () {
+  declare -r version="${1}";
+
+  declare -r installer_path="racket-minimal-${version}-x86_64-linux-natipkg.sh";
+  declare -r installer=$(installer_url "${version}" "${installer_path}") || exit "${?}";
+  build "racket" "buildpack-deps:stable" "${installer}" "${version}" "${USERNAME}/racket:${version}";
+
+  declare -r cs_installer_path="racket-minimal-${version}-x86_64-linux-natipkg-cs.sh";
+  declare -r cs_installer=$(installer_url "${version}" "${cs_installer_path}") || exit "${?}";
+  build "racket" "buildpack-deps:stable" "${cs_installer}" "${version}" "${USERNAME}/racket:${version}-cs";
+
+  declare -r full_installer_path="racket-${version}-x86_64-linux-natipkg.sh";
+  declare -r full_installer=$(installer_url "${version}" "${full_installer_path}") || exit "${?}";
+  build "racket" "buildpack-deps:stable" "${full_installer}" "${version}" "${USERNAME}/racket:${version}-full";
+
+  declare -r full_cs_installer_path="racket-${version}-x86_64-linux-natipkg-cs.sh";
+  declare -r full_cs_installer=$(installer_url "${version}" "${full_cs_installer_path}") || exit "${?}";
+  build "racket" "buildpack-deps:stable" "${full_cs_installer}" "${version}" "${USERNAME}/racket:${version}-cs-full";
+};
+
 build_6x_7x () {
```

Comment:
Could we change this function's name to `build_6x_7x_old` to make it clearer that only the first few 7x versions are built by this function?